### ISSUE TITLE
Replace <name> by <hi> in XML sources

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-gen-html#46]

**Comment by gerritbruening** [comment]
> Bare `<hi>` find ich komisch

Ja, ein minimaler Informationsverlust, aber mir fällt kein Attribut ein.

**Comment by thvitt** [comment]
Ich versteh das Problem inhaltlich nicht. 
- Es gibt einen Haufen `<hi status="name">`, mit dem wohl Figurennamen codiert sind, was aber nicht valide ist und laut Wiki eine Krücke für `<name>`. Ist hier immer etwas hervorgehoben, und es handelt sich immer um einen Figurennamen, aber es ist nicht codiert, wie die Hervorhebung erfolgt?  
- Die ex. `<name>` sollen `<hi>` werden. Ist hier überhaupt etwas hervorgehoben?
- Warum soll die ex. Figurennamenauszeichnung weg? Beim konstituierten Text wurde da doch ziemlicher Aufwand hineingesteckt? Weil nicht alle Figurennamen hervorgehoben sind? Weil bei `<name>` nicht immer hervorgehoben ist?
- Wenn es hervorgehobene Figurennamen sind, aber die Art der Hervorhebung nicht codiert ist, vielleicht `<name rend="highlighted">` oder so? rend gibt’s an name, aber keine Semantik an `<hi>`?

M.E. sollte die Entscheidung bzgl. `<name>` und `<hi status="name">` jedenfalls gemeinsam getroffen werden …

**Comment by gerritbruening** [comment]
Kurze Version: `hi` ist richtig.

Lange Version:
* `name` ist eine Kodierung aus unserer Frühzeit 2009/10
* `hi status="name"` ein späterer Ersatz dafür
* beides nicht konsistent über den gesamten Datenbestand
* `stage/hi` ist praktisch immer die Hervorhebung von Figurennamen, deswegen wurde der diesbezügliche Aufwand für die Druckausgabe faktisch komplett in `stage/hi` gesteckt, außer diesem gab es nur
* `stage/emph` (Handvoll Spezial-Fälle, anderes Issue)

**Comment by thvitt** [comment]
also sowohl `name` als auch `hi status="name"` zu `hi` mit kopierten Attributen. Ok.

**Comment by gerritbruening** [comment]
Heißt, `status="name"` wird mitkopiert?
Dann bei `name` ergänzen, wäre konsequent.

**Comment by thvitt** [comment]
nö, weggeworfen. Alle anderen Attribute werden kopiert.

**Comment by gerritbruening** [comment]
alles klar, auch gut

**Comment by gerritbruening** [comment]
I still find `attribute "status" not allowed here` triggered by `<hi status="name">Deputation der Gnomen</hi>` and the like.
